### PR TITLE
Apply WP coding standards to sync code

### PR DESCRIPTION
**Pull request by lezama**
No logic should have changed with this.

### CODE REVIEW

**Comment by lezama** [comment]
cc @gravityrail

File: sync/class.jetpack-sync-module-posts.php [hunk 1]
```diff
@@ -57,16 +59,17 @@ function filter_post_content_and_add_links( $post_object ) {
 		 * other services.
 		 * @since 4.2.0
 		 *
-		 * @param boolean false prevent post data from bing sycned to WordPress.com
+		 * @param boolean false prevent post data from bing synced to WordPress.com
```
Comment (gravityrail) [inline sync/class.jetpack-sync-module-posts.php]: bing -> being

File: sync/class.jetpack-sync-wp-replicastore.php [hunk 2]
```diff
@@ -42,27 +42,27 @@ function full_sync_end( $checksum ) {
...
+		if ( null != $max_id ) {
+			$where .= ' AND ID <= ' . intval( $max_id );
 		}
 
 		return $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->posts WHERE $where" );
 	}
 
 	public function get_posts( $status = null, $min_id = null, $max_id = null ) {
-		$args = array( 'orderby' => 'ID', 'posts_per_page' => -1 );
+		$args = array( 'orderby' => 'ID', 'posts_per_page' => - 1 );
```
Comment (gravityrail) [inline sync/class.jetpack-sync-wp-replicastore.php]: Is it really the standard to have a space between the sign of an integer and the number itself?

**Comment by gravityrail** [comment]
Looks great! Beautiful improvement ☀️

**Pull request merged**